code/modules/virus2/effect/stage_3.dm

```diff
@@ -449,3 +449,22 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
 		E.add_child(EE)
 		E.throw_at(get_step(src,pick(alldirs)), rand(1,4), rand(1,3))
 	..()
+
+
+/datum/disease2/effect/catvision
+	name = "Cattulism Syndrome"
+	stage = 3
+	max_count = 9
+	chance = 10
+	max_chance = 15
+
+/datum/disease2/effect/catvision/activate(var/mob/living/carbon/mob)
+	if (mob.see_in_dark_override != mob.see_in_dark)
```

Comment:
This is just going to keep resetting the mob.see_in_dark_override

Reply:
oh, good catch. i must've wrote that at about 5 am. the idea was to set the override to the mobs current see_in_dark value instead of the initial override value of 0. dont know why i did it like that.